### PR TITLE
Drawer

### PR DESCRIPTION
**Pull request by Aman-Jain1996**
- Added Drawer Component

Major file changes - 

drawer -> drawer.html
            -> drawer.css
            -> drawer.js

### CODE REVIEW

File: components/drawer/drawer.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  document
+    .querySelector(".dismissible-main-content")
+    .classList.remove("expanded-main-content");
+  menuIcon.style.display = "none";
+}
+
+menuIcon.addEventListener("click", showMenu);
+
+function HideMenu() {
+  console.log();
```
Comment (Rohanmond) [inline components/drawer/drawer.js:18]: remove this console.log

**Review by Rohanmond — APPROVED** [review]
good folder structure, code is clean, liked the way you have used drawar

File: components/drawer/drawer.js (hunk 1 above)
Comment (Aman-Jain1996) [reply components/drawer/drawer.js:18]: Thanks Rohan for review , will do that

**Review by Meghana-sk — APPROVED** [review]
Clean code. Good class naming conventions used. Functionality of drawer working as expected. Good job Aman.

**Pull request merged**